core/src/test/java/com/google/errorprone/bugpatterns/StatementSwitchToExpressionSwitchTest.java

```diff
@@ -1030,6 +1030,25 @@ public void emptySwitchCases_noMatch() {
             "    }",
             "  }",
             "}")
+        .setArgs("-XepOpt:StatementSwitchToExpressionSwitch:EnableDirectConversion")
```

Comment:
Added this for symmetry with other tests, and because in theory it covers more code paths (except that in practice `isRuleSwitch(switchTree)` is tested before the flag is consulted).

NB: the other tests use `.setArgs(ImmutableList.of(...))`; I wouldn't mind opening a separate cleanup PR for that.